test/fixture/apache-index.html

```diff
@@ -40,14 +40,14 @@
     <td valign="top"><img src="/icons/folder.gif" alt="[]"></td>
     <td><a href="folderB/">folderB/</a></td>
     <td align="right">19-May-2016 17:57</td>
-    <td align="right"> 5.4M</td>
+    <td align="right"></td>
     <td>&nbsp;</td>
   </tr>
   <tr>
     <td valign="top"><img src="/icons/compressed.gif" alt="[   ]"></td>
     <td><a href="fileA.zip">fileA.zip</a></td>
     <td align="right">28-May-2009 15:09</td>
-    <td align="right"> 56K</td>
+    <td align="right"> 5.4M</td>
```

Comment:
If I remember correctly, I removed the `size` field from `folderB/` (folders shouldn't have a size). But I still wanted to test that we are parsing `M` (megabyte) values correctly.